src/components/HeadingSettings/HeadingSettings.tsx

```diff
@@ -58,12 +67,58 @@ const HeadingSettings: FC<Props> = ({
         )}
       >
         <SettingsContainer isDisabled={!!settingsDisabled}>
+          {savedQueryName && (
+            <MousePositionedTooltip
+              isActive={!!savedQueryName}
```

Comment:
If I understand correctly savedQueryName is always true, so maybe setting _isActive_ without condition will be enough

Reply:
As discussed - the savedQueryName is not always `true` as user could create a ad-hoc query in Dashboard Creator or Explorer.